vcd/resource_vcd_org_vdc_test.go

```diff
@@ -40,13 +40,13 @@ func TestAccVcdOrgVdcReservationPool(t *testing.T) {
 		"CpuGuaranteed":    "1",
 		// The parameters below are for Flex allocation model
 		// Part of HCL is created dynamically and with empty values we don't create the Flex part:
-		"equalsChar":                         "",
-		"FlexElasticKey":                     "",
-		"FlexElasticValue":                   "",
-		"FlexElasticValueUpdate":             "",
-		"FlexMemoryOverheadKey":              "",
-		"FlexMemoryOverheadValue":            "",
-		"FlexMemoryOverheadValueUpdate":      "",
+		"equalsChar":                         " ",
```

Comment:
I guess that the reason for this change is to trick `testParamsNotEmpty` that only checks for empty strings. 
If this is the case, I question the validity of that function. Anyway, a comment to justify this change from empty string to space is in order.

Reply:
I didn't want to hide specific cases' check in function.